lib/PrintAsClang/PrintSwiftToClangCoreScaffold.cpp

```diff
@@ -239,7 +219,7 @@ void swift::printSwiftToClangCoreScaffold(SwiftToClangInteropContext &ctx,
                                             /*isCForwardDefinition=*/true);
           });
           os << "\n";
-          printCxxNaiveException(os);
+          //printCxxNaiveException(os);
```

Comment:
Is that intentional?

Reply:
No! Thanks for pointing that out!

Reply:
Fixed on https://github.com/apple/swift/pull/62197/commits/bfc8655ca4372710bf09468bdb03850407de9cd5